src/ImageGallery.jsx

```diff
@@ -931,15 +931,25 @@ export default class ImageGallery extends React.Component {
       }
 
       if (this.props.showBullets) {
+        const bulletOnClick = event => {
+          if(item.bulletOnClick){
+            const result = item.bulletOnClick({item, itemIndex: index, currentIndex});
+            if(result === false){
```

Comment:
This seems like a specific use case, can you explain why you need a bullet click that does not slide?

Reply:
if we have a handler to bullet click already, we might as well add the option to return false from it to prevent scrolling for whatever reason.

id you want, i can indeed delete it.

Reply:
Yes, lets leave it as `item.bulletOnClick({item, itemIndex: index, currentIndex});` for now.